Create solicitud-lecciones

jenniferisasi opened the pull request:
Creating a file for lesson request in Spanish per #1232 

La idea es publicar la solicitud para mayo.

jenniferisasi committed:
Create solicitud-lecciones

Creating a file for lesson request in Spanish.

mariajoafana commented:
Tutorial sobre cómo construir conjuntos de datos para investigación en humanidades. Potencialmente yo podría escribir este tutorial durante mi sabático

mariajoafana commented:
Tutoriales de QGIS con la versión 3.4.5 y de Python3. La idea es mantener los tutoriales existentes con versiones anteriores y contribuir con nuevas con actualizaciones

arojascastro commented:
Otra propuesta:

**Uso de base de datos XML para publicar ediciones digitales**
Las ediciones académicas digitales suelen modelarse como documentos XML por lo que el uso de base de datos XML parece lógico para desarrollar el *back end* de una plataforma web. Sin embargo, implementar la infraestructura para que se ajuste a las necesidades de cada proyecto puede ser desconcertante debido a la escasez de documentación. Siguiendo con nuestro espíritu abierto, nos gustaría publicar un tutorial que explique de manera detallada cómo utilizar una base de datos XML (por ejemplo, eXistDB o BaseX) para publicar en línea una edición académica o un corpus de textos.

arojascastro commented:
Otra propuesta: 

**Estadística para humanistas**
La digitalización del patrimonio textual y el acceso a herramientas gratuitas facilitan el análisis cuantitativo de textos en formato digital; sin embargo, cuando llega el momento de interpretar los datos obtenidos, a menudo, los humanistas desconocen muchas de las nociones fundamentales de la estadística. Por este motivo, buscamos un tutorial en el que se explique el rol de la estadística en las ciencias, la preparación de un corpus de textos representativo, y, sobre todo, que cubra las operaciones matemáticas más habituales (media, frecuencia absoluta y relativa, dispersión, diversidad, *chi-squared test*, correlación, palabras clave, colocaciones, etc.) con independencia del lenguaje de programación o la herramienta utilizados por el usuario/a.

arojascastro commented:
Otra propuesta para principiantes:

**Metadatos para todos**
El término *metadatos* suele definirse como los *datos sobre los datos*, es decir, los datos que describen un recurso u objeto digital desde un punto de vista contextual, intelectual o estructural. Aprender a describir siguiendo estándares utilizados en bibliotecas, archivos y museos es una tarea fundamental para catalogar, clasificar e indexar. Sin embargo, los estándares existentes son muy numerosos y variados, por lo que se hace difícil tomar una decisión al iniciar un proyecto de digitalización. Por esta razón, nos gustaría publicar un tutorial que ofrezca una panorámica de los estándares existentes (MARC, EAD, DublinCore, LIDO, etc.), que compare su estructura y que ejemplifique su implementación con ejercicios a desarrollar por el usuario/a.

vgayolrs commented:
@mariajoafana coincido contigo en el asunto de mantener las lecciones anteriores, pero en el caso de Python original es necesario crear un pequeño tutorial que explique las diferencias de sintaxis entre P2 y P3. Y esto es urgente porque el asunto es que ya no está disponible la descarga de la versión P2.x con la que se escribieron los tutoriales originales. Entonces, es necesario indicar cuáles son las nuevas maneras de hacer en P3 lo que se hacía antes. Sería como un mini tutorial adjunto. Es muy sencillo. con solamente adaptar las recomendaciones de Gibs y Milligan que cité en #1273 , y se pueden ver en #407 y #408 
Por otro lado, es necesaria una lección sobre el uso de unicode en Python y que haya un enlace desde las lecciones originales, sobre todo las traducidas al español y –quizá pronto– al francés, porque cuando se utilizan textos en un idioma distinto al inglés las cosas se complican. Por ejemplo, al seguir el tutorial original con textos en español, mis resultados fueron estos:
<img width="136" alt="Captura de Pantalla 2019-04-25 a la(s) 22 45 45" src="https://user-images.githubusercontent.com/17665201/56782315-33a50780-67ac-11e9-92e7-971287060573.png">
Las "\xc3" y "\xe2" son indicaciones de que ahí hay un código distinto a ASCII y que Python no puede resolver si no se le da la orden correspondiente.
Un tutorial en ese sentido va a ser un texto de referencia no solamente en el ámbito de las HD.

arojascastro commented:
Gracias @vgayolrs por la explicación. A mí me parece muy grave el asunto (me inclinaría por archivar esas lecciones) y creo que vale la pena pedir una lección que cubra las diferencias entre una versión y otra y que explique como lidiar con lenguajes no ASCII. ¿Podrías escribir un párrafo resumiendo lo que has expuesto pero para que sea publicable en el página de Solicitud de lecciones?

vgayolrs commented:
@arojascastro , no es tan grave. Si bien la gente ya no va a encontrar Python 2 disponible, las lecciones se pueden seguir sin problema haciendo una pequeña lección o anotación que explique que, si se usa Python 3.x, hay que cambiar algunas pequeñas cosas en varias líneas, como lo sugirió Ian en su momento. Son lecciones muy buenas.
Por otro lado, claro que hago el resumen. Sin embargo, si les parece y dado que es algo que tengo que hacer para mi investigación en curso sobre el corpus jurídico castellano-indiano, creo que yo podria escribir tanto la nota como una lección sobre cómo obtener frecuencias de palabras y concordancias con Python a partir de textos con caracteres propios de otros idiomas distintos al inglés (unicode).

jenniferisasi committed:
Update solicitud-lecciones

I've added three ideas for lessons.

jenniferisasi committed:
Update solicitud-lecciones

Added an idea about a lesson for audio analysis (from the English lesson request).

jenniferisasi committed:
Update solicitud-lecciones

fixed error for Travis

jenniferisasi committed:
Update solicitud-lecciones

jenniferisasi committed:
Update snippets.yml

Add link to solicitud de lecciones or lesson request for Spanish.

jenniferisasi commented:
Sorry @mdlincoln, is Travis failing on this one because of the links HTML Proofer is finding about other lessons? (see #1289)

mdlincoln committed:
Merge branch 'gh-pages' into issue-#1232

mdlincoln commented:
@jenniferisasi I fixed the broken links and added the fixes to this branch.

jenniferisasi commented:
Thanks @mdlincoln! I'm going now to wait for a final review and approval by @mariajoafana

jenniferisasi commented:
I have to make a final change, as the page downloads for the lack of .md at the end of the name, my bad.
